.github/dependabot.yaml

```diff
@@ -62,3 +62,43 @@ updates:
   commit-message:
     prefix: "deps"
   open-pull-requests-limit: 10
+- package-ecosystem: "gomod"
+  directory: "/"
```

Comment:
curious the logic for having the same line for v1.5 for both `/` and `/azure-ipam`? Assuming the former will cover all directories?

Reply:
no, this line tells dependabot which directory to look in for the `go.mod`. We need it to update both `/go.mod` and `/azure-ipam/go.mod`